### PR TITLE
(Feature) Apply new design to step 5: "Crowdsale page"

### LINKED ISSUE #1089: Apply new design to Step 5 - CrowdsalePage
Child of #1082

### PR DESCRIPTION
**Pull request by gabitoesmiapodo**
Closes #1089 

**Description**

Implementation of new styles for Step 5: "Crowdsale page"

**Some screenshots:**

screen shot 2018-10-31 at 15 12 58
screen shot 2018-10-31 at 15 13 09
screen shot 2018-10-31 at 15 13 18
screen shot 2018-10-31 at 15 13 27

screen shot 2018-10-31 at 15 06 05
screen shot 2018-10-31 at 15 06 21
screen shot 2018-10-31 at 15 06 32
screen shot 2018-10-31 at 15 06 40

### CODE REVIEW

**Comment by dennis00010011b** [comment]
@gabitoesmiapodo 
App failed to compile on MacOS
I think it is problem with letter case sensitivity  for paths

screen shot 2018-10-31 at 12 38 09 pm

screen shot 2018-10-31 at 12 38 55 pm

**Comment by gabitoesmiapodo** [comment]
@dennis00010011b before I proceed with this, can you have a look at this? It's a fix for Git case sensitivity on MacOs (I had to use it not long ago): https://coderwall.com/p/5w3jca/git-case-sensitivity-on-os-x

I'm currently running this branch on MacOS and file paths and names seem to be OK.

screen shot 2018-10-31 at 17 04 23

**Comment by dennis00010011b** [comment]
> @dennis00010011b before I proceed with this, can you have a look at this? It's a fix for Git case sensitivity on MacOs (I had to use it not long ago): https://coderwall.com/p/5w3jca/git-case-sensitivity-on-os-x

@gabitoesmiapodo 
command `git config core.ignorecase true` doesn't work for me in this case. 
Build is succefully if directory renamed from 'src/components/crowdsale' to 'src/components/Crowdsale'

**Comment by dennis00010011b** [comment]
@gabitoesmiapodo 
Would be usable to add icon `copy` for Crowdsale Proxy Address field

screen shot 2018-10-31 at 2 25 06 pm

**Comment by gabitoesmiapodo** [comment]
@dennis00010011b 

`- Would be usable to add icon copy for Crowdsale Proxy Address field`

Sure, done.

**Comment by dennis00010011b** [comment]
@gabitoesmiapodo 
Publish page formatted in wrong way
screen shot 2018-11-01 at 4 11 46 pm

**Comment by gabitoesmiapodo** [comment]
@dennis00010011b 

That was changed in Step 4's branch. Brought them to this branch anyway, it should look fine now.

File: src/components/Crowdsale/CrowdsaleSummary.js [hunk 1]
```diff
@@ -0,0 +1,35 @@
...
+  contributorsCount,
+  currentRatePerETH,
+  endRatePerETH,
+  extraClassName = '',
+  isMintedCappedCrowdsale,
+  startRatePerETH,
+  tokensClaimed,
+  totalSupply
+}) => {
+  const isDutchAuctionCrowdsale = !isMintedCappedCrowdsale
```
Comment (fernandomg) [inline src/components/Crowdsale/CrowdsaleSummary.js]: `!mintedCapped` isn't necessarily `dutchAuction`. Currently it is, but if new strategies will be added in the future we can just make it a bit more generic like passing `strategies` in the props. What do you think?

```suggestion
  const { isMintedCappedCrowdsale, isDutchAuction } = strategies
```

and in the `index.js` file do something like:

```
const strategies = { isMintedCappedCrowdsale, isDutchAuction }

// ...
<CrowdsaleSummary
  {...}
  strategies={strategies}
  {...}
/>
```

Comment (gabitoesmiapodo) [reply src/components/Crowdsale/CrowdsaleSummary.js]: Yeah, you are right, but I did something different anyway: I created a component for each strategy (_CrowdsaleSummaryMintedCapped_ and _CrowdsaleSummaryDutchAuction_), to make testing and programming a little easier.

**Pull request merged**